sql_generators/glean_usage/templates/events_first_seen_v1.query.sql

```diff
@@ -0,0 +1,140 @@
+{{ header }} -- the header doesn't seem to work properly
+
+{% raw %}
+{% if is_init() %}
+{% endraw %}
+with
+eventsstream as (
+    select
+    client_id,
+    profile_group_id,
+    sample_id,
+    event_category,
+    event_name,
+    `event`,
+    cast(null as string) as criteria,
+    min(submission_timestamp) as first_submission_timestamp,
+    min(event_timestamp) as first_event_timestamp,
+    min_by(event_extra, submission_timestamp) as event_extra,
+    min_by(app_version_major, submission_timestamp) as app_version_major,
+    min_by(normalized_channel, submission_timestamp) as normalized_channel,
+    min_by(normalized_country_code, submission_timestamp) as normalized_country_code,
+    min_by(normalized_os, submission_timestamp) as normalized_os,
+    min_by(normalized_os_version, submission_timestamp) as normalized_os_version,
+    from
+    `{{ project_id }}.{{ app_name }}_derived.{{ base_table }}`
+    where
+    -- initialize by looking over all of history
+    date(submission_timestamp) >= '2023-01-01'
+    and profile_group_id is not null
+    and event_category not in (
+      'media.playback',
+      'nimbus_events',
+      'uptake.remotecontent.result'
+    )
```

Comment:
If you add `sample_id = @sample_id`, it will run the init per sample id.  This is the only way to do this without the query timing out.
```suggestion
    )
    AND sample_id = @sample_id
```
It will block deployments for however many hours it runs so I would recommend either removing the is_init completely or making only run for the `MIN(DATE(submission_timestamp))`, and then rely on a managed backfill.

Reply:
As discussed, I'm removing the entire _is_init_ block and doing a managed backfill instead

Reply:
As discussed, we are reverting back to the _is_init_ block